_posts/2020-05-15-state-of-presto.md

```diff
@@ -26,6 +26,10 @@ Agenda:
 - Project roadmap
 - Live Q&A
 
-> ## [Register now!](https://www.starburstdata.com/webinar-state-of-presto/)
+Date: Thursday, 21 May 2020
+
+Time: 11 am PST (San Francisco), 1pm EST (New York), 7pm BST (London), 3am JST (Tokyo)
```

Comment:
These US time zones are incorrect, since we're in daylight savings now, so they should be PDT and EDT.

Reply:
Also, the Eastern time seems wrong, since it should be 2pm.

Reply:
Pushed an update.. I think it should be correc tnow